docs/sources/flow/reference/components/pyroscope.ebpf.md

```diff
@@ -0,0 +1,246 @@
+---
+title: pyroscope.ebpf
+labels:
+  stage: beta
+---
+
+# pyroscope.ebpf
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`pyroscope.ebpf` configures an ebpf profiling job for the current host. The collected performance profiles are forwarded
+to
+the list of receivers passed in
+`forward_to`.
+
+You can specify multiple `pyroscope.ebpf` components by giving them different labels, however it is not recommended.
```

Comment:
Can you expand on why it's not recommended in the documentation? Users might accidentally end up in a situation where they have more than one pyroscope.ebpf component if they are using two different external modules which both happen to use this component.

Reply:
```
You can specify multiple `pyroscope.ebpf` components by giving them different labels, however it is not recommended as
it can lead to additional memory and CPU usage.
```
Do you think it is enough?